recipe/meta.yaml

```
@@ -1,56 +1,55 @@
 {% set name = "spacy-transformers" %}
 {% set module_name = "spacy_transformers" %}
-{% set version = "1.2.4" %}
+{% set version = "1.3.5" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 859c2093fad9ff41005bc55c0018d4290bdd62454f76bddbcf318aae85f307b5
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: accdfe44a26517714c6990ec6bae88796eb348286fea7ba20ba2736f70c83fa1
+
 
 build:
   number: 0
-  # spacy isn't available on s390x
-  skip: True  # [py<36 or s390x]
-  # At present, there are no 3.11 compatible Pytorch version for win or ppc
-  # There is a bug requiring PyTorch 1.10 on osx-64 which is not available for 3.11
-  # As transformers 4.29.2 b0 has the same issue, we apply its skipping logic here  
-  skip: True  # [py>310 and (ppc64le or win or (osx and x86_64))]
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  # spacy isn't available on s390x.
+  # transformers isn't available for py>=312 currently.
+  skip: True  # [s390x or py>311]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   build:
     - {{ compiler('cxx') }}
   host:
     - python
-    - cython 0.29.32
+    - cython >=0.25
     - numpy {{ numpy }}
     - pip
     - setuptools
     - wheel
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    # TODO: Working around osx bug importing two openmp variants.
-    - pytorch >=1.8.0     # [not (osx and x86_64)]
-    - pytorch >1.9,<1.12  # [osx and x86_64]
-    - spacy >=3.5.0,<4.0.0
-    - spacy-alignments >=0.7.2,<1.0.0
+    - spacy >=3.5.0,<4.1.0
+    # transformers[sentencepiece] means 'extras["sentencepiece"] = deps_list("sentencepiece", "protobuf")',
+    # see https://github.com/explosion/spacy-transformers/blob/v1.3.5/requirements.txt#L3-L3C28 and
+    # https://github.com/huggingface/transformers/blob/345b9b1a6a308a1fa6559251eb33ead2211240ac/setup.py#L300-L300C65
+    # Anaconda set the transformers upper bound to <4.42.0 because of compatibility with 'tokenisers' versions
+    - transformers >=3.4.0,<4.42.0
+    - sentencepiece >=0.1.91,!=0.1.92
+    - protobuf
```

I don't see requirements.txt actually being used. We should follow https://github.com/explosion/spacy-transformers/blob/v1.3.5/setup.cfg#L32-L40
Perhaps what is in requirements.txt is useful for tests though.
```suggestion
    - transformers >=3.4.0,<4.37.0
```

---

It fails if we use `transformers >=3.4.0,<4.37.0` because the old version of transformers is picked up and then the incompatible version tokenizers will be used. I can update transformers but I haven't evaluated yet how much time it will take

---

Ok, you can use this as a patch
https://github.com/explosion/spacy-transformers/pull/418.patch and re-enable pip check.
(The change is compatible: https://github.com/explosion/spacy-transformers/compare/v1.3.5..master)
But I would still remove sentencepiece and protobuf which I don't see being used in setup.cfg.

---

Done. But there was an issue that `setup.cfg` on **PyPI** is not the same as on the **GitHub**. I temporarily replaced `source/url`